evm-contracts/src/v0.6/dev/PrepaidAggregator.sol

```diff
@@ -84,6 +84,7 @@ contract PrepaidAggregator is AggregatorInterface, Owned, WithdrawalInterface {
     uint32 indexed round,
     address indexed oracle
   );
+  event NewRound(uint256 indexed roundId, address indexed startedBy, uint256 startedAt);
```

Comment:
I think we should update the NewRound event in AggregatorInterface rather than adding a new event here.

Reply:
@thodges-gh okay I'll make that change